src/components/City/SensorsQualityStats/DataTable.js

```diff
@@ -127,10 +127,7 @@ PMValue.propTypes = {
 function HumidityValue({ classes, value }) {
   return (
     <DataValue value={value} classes={classes}>
-      <small className={classes.small}>
-        g/m
-        <sup>3</sup>
-      </small>
+      <small className={classes.small}>% RH</small>
```

Comment:
We've been displaying wrong unit all this time?!!!... cc @nerdlet / @karimkawambwa

Reply:
Sadly i have never looked into the units event of the hardware. I haven't even learnt the accuracy and thresholds of these device readings, I should read up on them.